railties/lib/rails/commands/credentials/credentials_command.rb

```diff
@@ -52,7 +57,6 @@ def change_credentials_in_system_editor
           end
         end
 
-
```

Comment:
Separation of method concerns. Just leave it in, thanks.

Reply:
Still needs to be left in.

Reply:
What do you think is needed? Seperate conditions in `#ensure_master_key_has_been_added` out here?